template/content/clients.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
```

Comment:
Might be worth considering renaming this to `deps.js`. "Clients" gets a bit ambiguous especially with things like loggers, etc. You typically dont think about your logger as a "client" to the logging subsystem, but you do think of a logger as a dependency of your service. Food for thought...